aws/resource_aws_dynamodb_table.go

```diff
@@ -793,7 +796,7 @@ func readDynamoDbTableTags(arn string, conn *dynamodb.DynamoDB) (map[string]stri
 		return nil, fmt.Errorf("Error reading tags from dynamodb resource: %s", err)
 	}
 
-	result := tagsToMapDynamoDb(output.Tags)
+	result := keyvaluetags.DynamodbKeyValueTags(output.Tags)
```

Comment:
When reading tags from the AWS APIs, we should ignore AWS system tags 👍 Also suggesting the call to `.Map()` to conform with the old function signature as mentioned above.

```suggestion
	result := keyvaluetags.DynamodbKeyValueTags(output.Tags).IgnoreAws().Map()
```

See also the original implementation in `tagsToMapDynamoDb()` that called `tagIgnoredDynamoDb()`  🎉